public/components/overview/office-panel/config/module-config.tsx

```diff
@@ -35,5 +35,17 @@ export const ModuleConfig = {
       <OfficeDrilldown title={'Client IP'} {...{ ...drilldownIPConfig, ...props }} />
     ),
   },
+  'data.office365.Operation': {
+    length: () => drilldownOperationsConfig.rows.reduce((total, row) => total + row.columns.length, 0),
+    component: (props) => (
+      <OfficeDrilldown title={'Operation'} {...{ ...drilldownOperationsConfig, ...props }} />
+    ),
+  },
+  'rule.description': {
+    length: () => drilldownOperationsConfig.rows.reduce((total, row) => total + row.columns.length, 0),
+    component: (props) => (
+      <OfficeDrilldown title={'Rule'} {...{ ...drilldownRulesConfig, ...props }} />
+    ),
```

Comment:
**thought**: I am not sure why we would want a view related to the `rule.description`.

Reply:
It is one among the few properties that are present across all alerts, and while adding a table for Operations I realized that there were cases with several operations to a single rule, so I thought it could be useful in a first iteration to show rules themselves.